src/Button.tsx

```diff
@@ -0,0 +1,22 @@
+function Button({
+  handleClick,
+  label,
+  autofocus = false,
+}: {
+  handleClick: Function;
```

Comment:
If you want to follow the conventions of built-in components, callback props should have `on` as their prefix

also, `Function` is a slightly evil TypeScript type because it doesn't enforce the function interface at all

```suggestion
  onClick: () => void;
```